src/Modules/Windows-Core+Full/CimCmdlets/CimCmdlets.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="{Fb6cc51d-c096-4b38-b78d-0fed6277096a}"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."